CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Change Log
 
+## 2.9.0
+- Added aggregate javadoc plugin
+[PR 242](https://github.com/aws/aws-xray-sdk-java/pull/242)
+- Configurtion to always create trace ID
+[PR 243](https://github.com/aws/aws-xray-sdk-java/pull/243)
+- Fix sampling rules not being deleted
+[PR 244](https://github.com/aws/aws-xray-sdk-java/pull/244)
+- Lock accesses to entities
+[PR 250](https://github.com/aws/aws-xray-sdk-java/pull/250)
+- Added support for reading header from environment and system properties
+[PR 252](https://github.com/aws/aws-xray-sdk-java/pull/252)
+- CodeCov report on pull requests
+[PR 261](https://github.com/aws/aws-xray-sdk-java/pull/261)
```

Comment:
Generally repo/operational improvements like this, smoke tests, and metrics shouldn't go in CHANGELOG since they aren't customer-facing features.